tests/docker-compose-cleanup.bats

```diff
@@ -4,19 +4,8 @@ load '/usr/local/lib/bats/load.bash'
 load '../lib/shared'
 load '../lib/run'
 
-@test "Default cleanup of docker-compose" {
-  run_docker_compose() {
-    echo "$@"
-  }
-  run compose_cleanup
-
-  assert_success
-  assert_equal "${lines[0]}" "kill"
-  assert_equal "${lines[1]}" "rm --force -v"
-  assert_equal "${lines[2]}" "down --volumes"
-}
-
```

Comment:
why did you remove this test?

Reply:
maybe I misunderstood, but I felt this behavior was already covered by `Cleanup runs after a run command` in `cleanup.bats`

Reply:
There is a lot of overlap in tests... but the one in the `cleanup.bats` file test the `pre-exit` hook script, these ones test the specific functions in the `lib/shared.bash` file instead. If the latter break, it is almost sure that the hook script that uses it will break, but the inverse is not true